geode-core/src/main/java/org/apache/geode/management/internal/cli/json/AbstractJSONFormatter.java

```diff
@@ -112,25 +127,16 @@ void postCreateMapper() {
       this.serializationDepth = serializationDepth;
     }
 
-    boolean isPrimitiveOrWrapper(Class<?> klass) {
-      return klass.isAssignableFrom(Byte.class) || klass.isAssignableFrom(byte.class)
-          || klass.isAssignableFrom(Short.class) || klass.isAssignableFrom(short.class)
-          || klass.isAssignableFrom(Integer.class) || klass.isAssignableFrom(int.class)
-          || klass.isAssignableFrom(Long.class) || klass.isAssignableFrom(long.class)
-          || klass.isAssignableFrom(Float.class) || klass.isAssignableFrom(float.class)
-          || klass.isAssignableFrom(Double.class) || klass.isAssignableFrom(double.class)
-          || klass.isAssignableFrom(Boolean.class) || klass.isAssignableFrom(boolean.class)
-          || klass.isAssignableFrom(String.class) || klass.isAssignableFrom(char.class)
-          || klass.isAssignableFrom(Character.class) || klass.isAssignableFrom(java.sql.Date.class)
-          || klass.isAssignableFrom(java.util.Date.class)
-          || klass.isAssignableFrom(java.math.BigDecimal.class);
+    private boolean isPrimitiveOrWrapper(Class<?> klass) {
+      return JsonUtil.isPrimitiveOrWrapper(klass);
```

Comment:
There is a slight change in that we used to allow  Date (sql and util) and Big Decimal objects.  I may have missed it but the new method didn't look like it checked for those class types.  Perhaps we were never using them in the first place?

Reply:
Hey @jhuynh1,

Thanks for reviewing this.
The classes `java.util.BigDecimal` and `java.util.Date` are automatically managed by `Jackson`, that's why we don't need to do anything special with them in the new version (other than include the type information as we do with the rest of the classes). Regarding `java.sql.Date`, it has a [custom serializer](https://github.com/apache/geode/pull/3688/files?file-filters%5B%5D=.java&owned-by%5B%5D=#diff-f42b9bfba8bf2ac20b7e93fe0910cb5bR168) registered within the [`postCreateMapper` method](https://github.com/apache/geode/pull/3688/files?file-filters%5B%5D=.java&owned-by%5B%5D=#diff-f42b9bfba8bf2ac20b7e93fe0910cb5bR77).
Cheers.